css/style.css

```diff
@@ -0,0 +1,71 @@
+body, p, div, header, footer {
+    margin: 0px;
+    padding: 0px;
+}
+div, header, p {
+    /*
+    border: 1px solid #2b2b2b;
+    */
+}
+
+.wrapper {
+    width: 1200px;
+    margin: auto;
+    font-size: 0;
+}
+.block {
+    height: 150px;
+    display: inline-block;
+    line-height: 150px;
+    vertical-align: middle;
```

Comment:
it's about `.block` height, it's the same (150) every time, so vertical-align doesn't make sense